linked-lists/tests/linkedList.test.js

```diff
@@ -228,6 +227,18 @@ describe('LinkedList class insertAfter method tests', () => {
   });
 });
 
+describe('LinkedList class kthFromEnd method tests', () => {

```

Comment:
I'm guessing you were planning to have more than one test inside this `describe` that uses a linked list of 2 nodes...
Since there is only one test here though, there is no need to create the variable outside the `test`. (The variable being `linkedList` that's been appended to twice.)